lib/index.js

```diff
@@ -19,7 +19,7 @@ var world;
 var overlord;
 var overlordImage;
 var trayCollision;
-
+// can add new game stuff into world or a renamed world (as game.js)
```

Comment:
Really like adding in these comments for potential refactoring so we can come back to this code for little nonconsecutive refactor sprints.